### ISSUE TITLE
Undebuggable VM extensions as scripts

### ISSUE THREAD

**Issue by jkanczler** [F5Networks/f5-azure-arm-templates#195]
## Do you already have an issue opened with F5 support?
No.

## Description
The supported ARM templates look unprofessional. Hard to debug. Executing the referenced ARM template gives the following error:

```
##[error]VMExtensionProvisioningError: VM has reported a failure when processing extension 'start'. Error message: "Enable failed: failed to execute command: command terminated with exit status=1
[stdout]

[stderr]
/bin/sh: -c: line 87: unexpected EOF while looking for matching `''
/bin/sh: -c: line 88: syntax error: unexpected end of file
"
```

Based on the above error, I would like to request a new feature from you. Next time, please use a professional solution to implement a debuggable approach, instead of concatenating magical scripts in ARM templates like this.

## Template
https://github.com/F5Networks/f5-azure-arm-templates/blob/master/supported/failover/same-net/via-lb/1nic/existing-stack/byol/azuredeploy.json

## Severity Level
5

**Comment by shyawnkarim** [comment]
Thanks for providing this feedback. We continually strive to improve our error messages but sometimes they can be quite ambiguous. I've added an enhancement to the backlog for this error and are tracking it internally with ID ESECLDTPLT-2344.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-azure-arm-templates-v2).
